src/Umbraco.Core/Cache/ObjectCacheAppCache.cs

```diff
@@ -34,22 +31,16 @@ public class ObjectCacheAppCache : IAppPolicyCache, IDisposable
     /// Initializes a new instance of the <see cref="ObjectCacheAppCache" />.
     /// </summary>
     public ObjectCacheAppCache()
-        : this(Options.Create(new MemoryCacheOptions()), NullLoggerFactory.Instance, null)
+        : this(new MemoryCacheOptions(), NullLoggerFactory.Instance)
```

Comment:
`MemoryCacheOptions` already implements `IOptions<MemoryCacheOptions>`, so there's no reason to use `Options.Create()` 🙌🏻